Gems/Atom/RHI/Vulkan/Code/Source/RHI/SwapChain.cpp

```diff
@@ -120,13 +120,17 @@ namespace AZ
 
         void SwapChain::ShutdownInternal()
         {
+            m_pendingRecreation = false;
+
             InvalidateNativeSwapChain();
             InvalidateSurface();
 
             m_presentationQueue = nullptr;
 
             m_swapchainNativeImages.clear();
             m_currentFrameContext = {};
+
+            ResourcePool::ShutdownInternal();
```

Comment:
I think instead of this you probably want to do Base::ShutdownInternal(). We should also add this to other backends too.

Reply:
Yeah. You are right.